src/bus/bus.c

```diff
@@ -107,17 +107,55 @@ static int bus_log_commit_policy(Bus *bus, const char *action, uint64_t sender_i
                             receiver_id);
         }
 
+        if (sender_names) {
+                if (sender_names->type == NAME_SET_TYPE_OWNER) {
+                        NameOwnership *ownership;
+                        size_t i = 0;
+
+                        c_rbtree_for_each_entry(ownership,
+                                                &sender_names->owner->ownership_tree,
+                                                owner_node)
+                                log_appendf(log,
+                                            "DBUS_BROKER_SENDER_WELL_KNOWN_NAME_%zu=%s\n",
+                                            i++, ownership->name->name);
+                } else if (sender_names->type == NAME_SET_TYPE_SNAPSHOT) {
+                        for (size_t i = 0; i < sender_names->snapshot->n_names; ++i)
+                                log_appendf(log,
+                                            "DBUS_BROKER_SENDER_WELL_KNOWN_NAME_%zu=%s\n",
+                                            i, sender_names->snapshot->names[i]->name);
+                }
+        }
+
+        if (receiver_names) {
+                if (receiver_names->type == NAME_SET_TYPE_OWNER) {
+                        NameOwnership *ownership;
+                        size_t i = 0;
+
+                        c_rbtree_for_each_entry(ownership,
+                                                &receiver_names->owner->ownership_tree,
+                                                owner_node)
+                                log_appendf(log,
+                                            "DBUS_BROKER_RECEIVER_WELL_KNOWN_NAME_%zu=%s\n",
+                                            i++, ownership->name->name);
+                } else if (receiver_names->type == NAME_SET_TYPE_SNAPSHOT) {
+                        for (size_t i = 0; i < receiver_names->snapshot->n_names; ++i)
+                                log_appendf(log,
+                                            "DBUS_BROKER_SENDER_WELL_KNOWN_NAME_%zu=%s\n",
```

Comment:
This must be `DBUS_BROKER_RECEIVER_WELL_KNOWN_NAME_`

Reply:
Will fix. Thanks!